nix/t1/conversion/sv-to-vcs-simulator.nix

```diff
@@ -22,7 +24,12 @@ stdenv.mkDerivation rec {
   dontPatchELF = true;
   enableCover = true;
 
-  src = rtl;
+  srcs = [
+    rtl
+    vsrc
```

Comment:
Since the following vcs command line args are supplied with full vsrc path. Do we need vsrc in source array?

Reply:
I'm not familiar with nix. But after removing 'srcs', it also works. Are you suggesting we should remove 'srcs' here?

Reply:
Just let src to be rtl should be OK.